src/services/calendar/index.js

```diff
@@ -45,7 +49,11 @@ const convertEventToJsonApi = (body) => ({
 				location: body.location,
 				description: body.description,
 				dtstart: body.startDate,
-				dtend: body.endDate || new Date(new Date(body.startDate).getTime() + body.duration).toISOString(),
+				dtend:
+					body.endDate ||
```

Comment:
Codacy found an issue: ['||' should be placed at the beginning of the line.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)